pkgs/development/python-modules/scikit-hep-testdata/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, importlib-resources
+, pyyaml
+, requests
+, pytestCheckHook
+, setuptools_scm
+, git
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-hep-testdata";
+  version = "0.4.2";
```

Comment:
In the meantime a [new release](https://github.com/scikit-hep/scikit-hep-testdata/releases/tag/v0.4.3) was published.